lib/workers/bin/run_single_worker.rb

```diff
@@ -60,27 +60,27 @@ def all_role_names
 opt_parser.parse!
 worker_class = ARGV[0]
 
-require File.expand_path("../miq_worker_types", __dir__)
+require File.expand_path("../../../config/environment", __dir__)
 
 if options[:list]
-  puts ::MIQ_WORKER_TYPES.keys
+  puts MiqWorkerType.pluck(:worker_type)
   exit
 end
 opt_parser.abort(opt_parser.help) unless worker_class
 
-unless ::MIQ_WORKER_TYPES.keys.include?(worker_class)
+worker_type = MiqWorkerType.find_by(:worker_type => worker_class)
+
+unless worker_type
   STDERR.puts "ERR:  `#{worker_class}` WORKER CLASS NOT FOUND!  Please run with `-l` to see possible worker class names."
   exit 1
 end
 
 # Skip heartbeating with single worker
 ENV["DISABLE_MIQ_WORKER_HEARTBEAT"] ||= options[:heartbeat] ? nil : '1'
-ENV["BUNDLER_GROUPS"] = MIQ_WORKER_TYPES[worker_class].join(',')
+ENV["BUNDLER_GROUPS"] = worker_type.bundler_groups.join(',')
```

Comment:
Commenting regarding discussion we had in person... because line 63 loads environment which will have already read and used the `BUNDLER_GROUPS` env var from application.rb [here](https://github.com/ManageIQ/manageiq/blob/3fa0b49815922a0eba4aa1f445567d9b6411dc34/config/application.rb#L21-L32), this manipulation will be too late.

Reply:
For this I moved the bundler groups envvar to the caller of run_single_worker.rb.

The logic was that we would either take the caller's env var if it was set and have some default in this script, but since that exact logic is already taken care of in [application.rb](https://github.com/ManageIQ/manageiq/blob/3fa0b49815922a0eba4aa1f445567d9b6411dc34/config/application.rb#L21), I didn't think it made sense to duplicate it here.

The only thing we lose with this approach is getting the correct bundler groups when someone runs run_single_worker.rb from the command line, but I would argue that if that person wants bundler groups, they will know to set the environment variable.